### PR TITLE
feat(): Npm publish private packages

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: src/api/routes.ts [hunk 1]
```diff
@@ -218,18 +218,38 @@ export function getTarball(req: auth.AuthRequest, res: express.Response) {
 }
 
 export function publishPackage(req: auth.AuthRequest, res: express.Response): void {
+  let jsonErrorResponse: any = { message: 'package published' };
```
Comment (jkuri) [inline src/api/routes.ts]: error response and `package published` message?

Comment (Izak88) [reply src/api/routes.ts]: oh yeah, stupid mistake... :)

Comment (Izak88) [reply src/api/routes.ts]: done

**Comment by jkuri** [comment]
merged with https://github.com/bleenco/morose/commit/fad17994ce00c0e38f129bcf13114542291ffb03

**Pull request closed without merging**